components/prize_pool/commons/prize_pool_placement.lua

```diff
@@ -374,6 +376,10 @@ function Placement:_getLpdbData(...)
 		lpdbData = Table.mergeInto(lpdbData, Opponent.toLpdbStruct(opponent.opponentData))
 
 		lpdbData.objectName = self.parent:_lpdbObjectName(lpdbData, ...)
+		if Opponent.isTbd(opponent.opponentData) then
+			_tbd_index = _tbd_index + 1
+			lpdbData.objectName = lpdbData.objectName .. _tbd_index
```

Comment:
```suggestion
			lpdbData.objectName = lpdbData.objectName .. '_' .. _tbd_index
```
To match the equivalent TeamCard entry
https://liquipedia.net/commons/index.php?title=Module:TeamCard/Storage&action=edit#mw-ce-l98